pages/lessons.html

```diff
@@ -39,7 +39,7 @@ <h4>Availability</h4>
   <div class="medium-4 columns">
     <h4>Contributing</h4>
     <p>
-      If you have questions about contributing to our lessons, visit each lesson's GitHub repo to submit an issue or to get the link to join that lesson's Maintainers' discussion on Slack. For general information on how to contribute to our lessons, see our <a href="https://github.com/swcarpentry/git-novice/blob/gh-pages/CONTRIBUTING.md">contributors guide</a>.  To learn more about how our lessons are structured, and why, please see <a href="{{site.carp_gh_io_url}}/lesson-example">the example lesson</a>. 
+      If you have questions about contributing to our lessons, visit each lesson's GitHub repo to submit an issue or to get the link to join that lesson's Maintainers' discussion on Slack. For general information on how to contribute to our lessons, see our <a href="https://github.com/swcarpentry/git-novice/blob/main/CONTRIBUTING.md">contributors guide</a>.  To learn more about how our lessons are structured, and why, please see <a href="{{site.carp_gh_io_url}}/lesson-example">the example lesson</a>. 
```

Comment:
We no longer use the `lesson-example` repository as the infrastructure now uses The Workbench.

```suggestion
      If you have questions about contributing to our lessons, visit each lesson's GitHub repo to submit an issue or to get the link to join that lesson's Maintainers' discussion on Slack. For general information on how to contribute to our lessons, see our <a href="https://github.com/swcarpentry/git-novice/blob/main/CONTRIBUTING.md">contributors guide</a>.  To learn more about how our lessons are structured, and why, please see <a href="{{site.carp_gh_io_url}}/workbench">The Workbench documentation and resources page</a>. 
```

Reply:
Thanks @zkamvar for the suggestion. I have committed the new changes.